docs/src/pages/backend-adapters/express.mdx

```diff
@@ -0,0 +1,166 @@
+
+{/* prettier-ignore-start */}
+
+<Tabs items={["npm", "yarn", "pnpm"]}>
+  <Tab>
+    ```bash copy 
+    npm install uploadthing
+    ```
+  </Tab>
+  <Tab>
+    ```bash copy 
+    yarn add uploadthing
+    ```
+  </Tab>
+  <Tab>
+    ```bash copy 
+    pnpm install uploadthing
+    ```
+  </Tab>
+</Tabs>
+
+{/* prettier-ignore-end */}
+
+### Add env variables
+
+```bash copy
+UPLOADTHING_SECRET=... # A secret key for your app (starts with sk_live_)
+UPLOADTHING_APP_ID=... # Your app id
+```
+
+<Callout type="warning">
+  If you don't already have a uploadthing secret key, [sign up](https://uploadthing.com/sign-in) and create one from
+  the [dashboard!](https://uploadthing.com/dashboard)
+</Callout>
+</Steps>
+
+## Set Up A FileRouter
+
+<Steps>
+
+### Creating your first FileRoute
+
+All files uploaded to uploadthing are associated with a FileRoute. Think of a
+FileRoute similar to an endpoint, it has:
+
+- Permitted types ["image", "video", etc]
+- Max file size
+- (Optional) `middleware` to authenticate and tag requests
+- `onUploadComplete` callback for when uploads are completed
+
+```ts copy filename="src/uploadthing.ts"
+import { createUploadthing } from "uploadthing/express";
+import type { FileRouter } from "uploadthing/express";
+
+const f = createUploadthing();
+
+export const uploadRouter = {
+  videoAndImage: f({
+    image: {
+      maxFileSize: "4MB",
+      maxFileCount: 4,
+    },
+    video: {
+      maxFileSize: "16MB",
+    },
+  })
+    .middleware(() => ({}))
+    .onUploadComplete((data) => {
+      console.log("upload completed", data);
+    }),
+
+  withMdwr: f({
+    image: {
+      maxFileCount: 2,
+      maxFileSize: "1MB",
+    },
+  })
+    .middleware((opts) => {
+      const h = opts.req.headers["someProperty"];
+
+      if (!h) throw new Error("someProperty is required");
+
+      return {
+        someProperty: h,
+        otherProperty: "hello" as const,
+      };
+    })
+    .onUploadComplete(({ metadata, file }) => {
+      console.log("uploaded with the following metadata:", metadata);
+      console.log("files successfully uploaded:", file);
+    }),
+
+  withoutMdwr: f({
+    image: {
+      maxFileCount: 2,
+      maxFileSize: "16MB",
+    },
+  })
+    .middleware(() => {
+      return { testMetadata: "lol" };
+    })
+    .onUploadComplete(({ metadata, file }) => {
+      console.log("uploaded with the following metadata:", metadata);
+      console.log("files successfully uploaded:", file);
+    }),
+} satisfies FileRouter;
+
+export type OurFileRouter = typeof uploadRouter;
+```
+
+### Create an API route using the FileRouter
+
+<Callout type="warning">
+  Note: You need to serve this API from `/api/uploadthing` on your application.
+</Callout>
+
+```ts copy filename="src/index.ts"
+import * as utExpress from "uploadthing/express";
+
+import { uploadRouter } from "./uploadthing";
+
+app.use(
+  "/api/uploadthing",
+  utExpress.createExpressRouter({
+    router: uploadRouter,
+  }),
+);
```

Comment:
we never use `import *` anywhere else

```suggestion
import { createExpressRouter } from "uploadthing/express";

import { uploadRouter } from "./uploadthing";

app.use(
  "/api/uploadthing",
  createExpressRouter({
    router: uploadRouter,
  }),
);
```

Reply:
I literally had the varian you proposed initially, but Mark asked to refactor so that our middleware is distinguishable from other ones that potentially may be present in code.

How about I rename the function to something like `createUTExpressRouter` and adjust import accordingly?